### PR TITLE
feat(wizards): Changed label of rptID

### LINKED ISSUE #636: Better description of rptID
The ID description is a bit limited:
image

Suggestion: ID (Report control block identifier)

### PR DESCRIPTION
**Pull request by dlabordus**
closes #636

### CODE REVIEW

File: src/translations/de.ts [hunk 1]
```diff
@@ -541,6 +541,7 @@ export const de: Translations = {
   },
   report: {
     wizard: { location: 'Ablageort der Reports wählen' },
+    rptID: '???'
```
Comment (JakobVogelsang) [inline src/translations/de.ts:544]:
```suggestion
    rptID: 'Report-Kontrolblock Kennung'
```

**Review by JakobVogelsang — CHANGES_REQUESTED** [review]
Other than the german translation very ok. Thanks for this!

File: src/translations/de.ts (hunk 1 above)
Comment (dlabordus) [reply src/translations/de.ts]: Fixed

**Review by Flurb — APPROVED** [review]

**Review by JakobVogelsang — APPROVED** [review]
LGTM

**Pull request merged**